scripts/pi-hole/php/teleporter.php

```diff
@@ -248,10 +279,10 @@ function flush_table($table, $type=null)
 	if(!in_array($table, $flushed_tables))
 	{
 		if($type !== null) {
-			$sql = "DELETE FROM ".$table." WHERE type = ".$type;
+			$sql = "DELETE FROM \"".$table."\" WHERE type = ".$type;
 			array_push($flushed_tables, $table.$type);
 		} else {
-			$sql = "DELETE FROM ".$table;
+			$sql = "DELETE FROM \"".$table."\"";
```

Comment:
Are the quotes and escapes correct? It looks like too many `"`s?

Reply:
Yes, this is correct. It's PHPs way to get:
```
DELETE FROM "table"
```

Reply:
So the end result is supported to be `"DELETE FROM ".$table." WHERE type = "` ?

Reply:
Sorry, I was looking at the wrong line. The end result should be
```
DELETE FROM "whatever" WHERE type = else
```
We need the extra `""` around the table name because `group` will otherwise result in an error (it is also a SQL keyword).

Reply:
Okay, sounds good.

Reply:
Would using single quotes around the table name improve readability? Not sure how `php` will see single quotes inside a quoted section..?